cylc/rose/utilities.py

```diff
@@ -189,10 +189,16 @@ def rose_config_exists(
 
     # If _any_ of the following are true we want to return True.
     if (
-        Path(srcdir, 'rose-suite.conf').is_file() or
-        opts and opts.opt_conf_keys or
-        opts and opts.defines or
-        opts and opts.rose_template_vars
+        (
+            opts and hasattr(opts, 'opt_conf_keys')
+            and opts.opt_conf_keys
```

Comment:
These can be simplifed to
```suggestion
            getattr(opts, 'opt_conf_keys', False)
```